spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -699,7 +755,7 @@ def main(argv: Sequence[str]):
             paramregmulti.steps[key].rot_dest = 0
         fname_src2dest, fname_dest2src, warp_forward, warp_inverse = register_wrapper(
             ftmp_data, ftmp_template, param, paramregmulti, fname_src_seg=ftmp_seg, fname_dest_seg=ftmp_template_seg,
-            same_space=True)
+            fname_src_label=ftmp_rootlet, fname_dest_label=ftmp_template_rootlets, same_space=True)
```

Comment:
I understand that `ftmp_rootlet` and `ftmp_template_rootlets` will be `None` if `-lrootlets` isn't passed. But, still, if someone is just skimming through the code, it might be a bit confusing to see these variables being passed unconditionally. ("I didn't use rootlets! Why is this being passed?" etc.)

So, maybe to better convey that the rootlets are completely optional, we could change it to this?

```suggestion
            fname_src_label=(ftmp_rootlet if label_type == 'rootlet' else None), 
            fname_dest_label=(ftmp_template_rootlets if label_type == 'rootlet' else None), 
            same_space=True)
```

It is technically redundant, but it at least self-documents that these options will only be used if `label_type == 'rootlet'`.

Reply:
sounds good!